study.md

```diff
@@ -43,5 +51,12 @@ your deployed Ember app? What do you need if you want to use the `history` API
 instead of `hash` for `location`?
 
 ```md
-<!-- your answer here -->
+
+A Hash URL is like `http://thingy.com/#/posts/1/history`. They have `#` in the URL.
+
+It's important not to prefix image data with a `/`. You must add an if statement saying if the environment is a production environment then change the base url to `/project-name` and set the locationtype to `hash`.
+You also need to be aware that you want to copy the compiled data from dist. then push to gh-pages.
+
+If you want to use the history api insted of a hash for location all you have to do is set `ENV.locationType` to `history`.
```

Comment:
You are correct that you would need to set the `locationType` to `history` instead of `auto` or `hash`.  If you want to use history API, then you can't deploy on github, you would need deploy your ember app to Heroku.